### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by aptvaibhav**

### CODE REVIEW

**Comment by aptvaibhav** [comment]
@bd808 could u please review this.

**Comment by aptvaibhav** [comment]
@yuvipanda please review.

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PAWS
+#PAWS
 
 
 A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface.
```
Comment (crookedstorm) [inline readme.md]: It would be more correct here to say "PAWS: A Web Shell (PAWS)" at the start of the first sentence, it's a recursive acronym like GNU.

Comment (aptvaibhav) [reply readme.md]: okay, thanks for reviewing.
Now could u review it again, i just completed your suggested changes. @crookedstorm

**Comment by crookedstorm** [comment]
Thank you for this! I looked at the full PR in terms of the file it would create, and I think there are a couple more things needed.

- filename: It should be README.md (all caps) to follow the usual convention https://en.wikipedia.org/wiki/README
- formatting: You need a space after the # in the first line, and there should be some sub-headings.  I'll make some suggestions inline.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+PAWS
+
+
+A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
+
+Quick Start
+
+git clone https://github.com/toolforge/paws.git
```
Comment (crookedstorm) [inline readme.md]: Enclose this in backtick characters "`" to make Markdown show it as code.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+cd paws
```
Comment (crookedstorm) [inline readme.md]: Same, use backticks

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Notebooks
+
+It provides Jupyter notebooks (previously known as IPython Notebooks)
+Web based Terminal
+Useful libraries
+Accessing Database Replicas With Pandas and Sqlalchemy
+
+Pandas is a lovely high level library for in-memory data manipulations. In order to get the result of a SQL query as a pandas dataframe use:
+
+from sqlalchemy import create_engine
```
Comment (crookedstorm) [inline readme.md]: For this entire code block and any others, please format with this:
https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks
You can even have the python code syntax highlighted with that feature.

**Comment by crookedstorm** [comment]
So aside from the inline comments, please make subheadings where you think it makes sense with "## title"

Then I think this will make a great start on in-repo documentation.

**Comment by aptvaibhav** [comment]
@crookedstorm could you please check if its now correct, Thank you.

**Comment by crookedstorm** [comment]
There are things we can improve on, but this is a great start that will make it a lot easier to improve going forward.  Thanks!

**Pull request merged**